app/src/org/commcare/android/tasks/FormRecordCleanupTask.java

```diff
@@ -251,12 +290,9 @@ public TransactionParser getParser(KXmlParser parser) {
         parsed.setID(r.getID());
 
         // Make sure that the instance is no longer editable
-        if (!newStatus.equals(FormRecord.STATUS_INCOMPLETE) &&
-                !newStatus.equals(FormRecord.STATUS_UNSTARTED)) {
-            ContentValues cv = new ContentValues();

```

Comment:
STATUS_SAVED was always passed as the value for newStatus, so we always want to execute the body of this conditional.